docs/features/recce-debug.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Debug
+icon: material/bug-check
+---
+
+The `recce debug` command is a diagnostic tool that helps you verify your Recce setup by checking the development and base environments. It validates that all required dbt artifacts are in place and tests the warehouse connection, ensuring everything is properly configured before running other Recce commands.
+
+## Usage
+
+```bash
+recce debug [OPTIONS]
+```
+
+## Options
+
+| Option                    | Description                                                       |
+| ------------------------- | ----------------------------------------------------------------- |
+| `-t, --target TEXT`       | Which target to load for the given profile                        |
+| `--profile TEXT`          | Which existing profile to load                                    |
+| `--project-dir PATH`      | Which directory to look in for the dbt_project.yml file           |
+| `--profiles-dir PATH`     | Which directory to look in for the profiles.yml file              |
+| `--target-path TEXT`      | dbt artifacts directory for your development branch               |
+| `--target-base-path TEXT` | dbt artifacts directory to be used as the base for the comparison |
+| `--help`                  | Show help message and exit                                        |
+
+## Example Outputs
+
+### Success Case
+
+When everything is properly configured, you'll see output like this:
+
+```
```

Comment:
You can add shell in Example Outputs. Optional since it only changes a few styling. 
![Screenshot 2025-06-17 at 5 43 33 PM](https://github.com/user-attachments/assets/9291fe6e-10f9-404d-9bf0-7915749ab229)

Reply:
I mean ```shell  instead of ```